main/res/values/strings.xml

```diff
@@ -791,6 +791,8 @@
     <string name="init_summary_bigSmileysOnMap">If enabled big emoticons will be shown for logged caches on the map instead of the cache type icon (requires restart).</string>
     <string name="init_global_wp_extraction_disable">Disable waypoint extraction</string>
     <string name="init_summary_global_wp_extraction_disable">Globally disable waypoint extraction from personal cache notes. It can still be enabled for individual caches by deactivating the checkbox below the personal cache note</string>
+    <string name="init_prevent_cache_note_merge">Prevent Merging of Personal Cache Notes</string>
+    <string name="init_summary_prevent_cache_note_merge">Prevent merging of personal cache note data from provider with locally stored personal caches notes data</string>
```

Comment:
Do you really mean "Do not merge" or do you mean "Replace local cache note by the server side cache note"?

Reply:
Interesting question. Hard to describe it with a sentence as brief as both variants are.

What is meant is the following: Upon initially loading a geocache object from the provider, take the personal cache notes from the provider unchanged. Further, upon refreshing a geocache object from the provider, take the personal cache notes from the provider unchanged. If a user has made local modifications to the personal notes fielld in cgeo he must push those modifications to the provider otherwise those changes are lost with the next refresh of the object.

Does `Replace local cache note by the server side cache note` transport these semantics sufficiently?